src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -365,6 +356,17 @@ public override void _Ready()
 
         mapDrawer.OnSelectedPatchChanged = (drawer) => { UpdateShownPatchDetails(); };
 
+        atpProductionBar = (SegmentedBar)SegmentedBarScene.Instance();
+        atpConsumptionBar = (SegmentedBar)SegmentedBarScene.Instance();
```

Comment:
I think that these could be already specified as scene instances in the editor scene.